gob/k8s/README.md

```diff
@@ -223,22 +223,22 @@ We'll deploy the _gen-growthhack_ pod and service (so that it can be
 distinguished from the prior version):
 
 ```
-:; kubectl apply -f k8s/gob/gen
+:; kubectl apply -f k8s/gob/gen-growthhack
```

Comment:
Based on the files you're adding, I think this should be:

```
:; kubectl apply -f k8s/gob/gen-growthhack
```

Also, can you remove the note about the growthhack branch that appears on line 211 in this file?  Maybe just change it to:

```
_A docker image with these changes is already published to
[gobsvc/gen:growthhack](https://hub.docker.com/r/gobsvc/gen/tags/)._
```